nucypher-core-wasm/src/lib.rs

```diff
@@ -1101,7 +1101,7 @@ impl ReencryptionResponse {
 
         Ok(Self(nucypher_core::ReencryptionResponse::new(
             signer.as_ref(),
-            backend_capsules.iter().zip(backend_vcfrags.into_iter()),
+            backend_capsules.iter().zip(backend_vcfrags),
```

Comment:
@piotr-roslaniec Let's be careful changing old PRE code unless necessary.

Is it that `into_iter()` no longer works, or is `zip(...)` just best practice?